pkg/reporter/stdout_reporter.go

```diff
@@ -94,7 +98,9 @@ func PrintTripleGroupStdout(groupReport map[string]map[string]map[string][]Repor
 				totalSuccessCount += stdoutReport.Summary.Passed
 				totalFailureCount += stdoutReport.Summary.Failed
 				fmt.Println(stdoutReport.Text)
-				fmt.Printf("        Summary: %d succeeded, %d failed\n\n", stdoutReport.Summary.Passed, stdoutReport.Summary.Failed)
+				if groupOne != "Passed" && groupOne != "Failed" && groupTwo != "Passed" && groupTwo != "Failed" && groupThree != "Passed" && groupThree != "Failed" {
+					fmt.Printf("        Summary: %d succeeded, %d failed\n\n", stdoutReport.Summary.Passed, stdoutReport.Summary.Failed)
+				}
```

Comment:
I would suggest you to create a method like this

```go
func checkGroup(groups ...string) {

	for , group := range groups {
		if group == "Passed" || group == "Failed" {
			return false
		}
	}

	return true
}
```

Then here, (and for the other checks), you can call


```suggestion
				if checkGroup(groupOne, groupTwo, groupThree) {
					fmt.Printf("        Summary: %d succeeded, %d failed\n\n", stdoutReport.Summary.Passed, stdoutReport.Summary.Failed)
				}
```

Reply:
Thanks for the feedback!

Reply:
Adding a Co-authored-by tag to your commit would be appreciated then

```
Co-authored-by: ccoVeille <3875889+ccoVeille@users.noreply.github.com>
```

https://docs.github.com/articles/creating-a-commit-with-multiple-authors

Reply:
Added the tag 👍